.github/workflows/native-gradle-plugin.yml

```diff
@@ -15,7 +15,21 @@ on:
 
 jobs:
   test-native-gradle-plugin:
-    name:
+    name: Sanity checks
```

Comment:
Sanity checks should use GraalVM (see the failed CI run [here](https://github.com/graalvm/native-build-tools/runs/3057775260?check_suite_focus=true)).

So it would be something like
```yaml
steps:
      - uses: actions/checkout@v2
      - name: Get GraalVM Nightly
        run: |
          source common/scripts/downloadGraalVM.sh
          echo "$GRAALVM_HOME/bin" >> $GITHUB_PATH
          echo "JAVA_HOME=$JAVA_HOME" >> $GITHUB_ENV
          echo "GRAALVM_HOME=$GRAALVM_HOME" >> $GITHUB_ENV
          ...
```

After that is fixed we could merge it for 0.9.3 since it would significantly cut down on CI execution time for Gradle.

Reply:
added